akka-docs-dev/rst/scala/http/routing-dsl/directives/file-and-resource-directives/getFromFile.rst

```diff
@@ -8,7 +8,7 @@ Completes GET requests with the content of the given file.
 Signature
 ---------
 
-.. includecode2:: /../../akka-http/src/main/scala/akka/http/server/directives/FileAndResourceDirectives.scala
+.. includecode2:: /../../akka-http-scala/src/main/scala/akka/http/scaladsl/server/directives/FileAndResourceDirectives.scala
    :snippet: getFromFile
 
 Description
```

Comment:
Add a note in the description on which dispatcher this will be run (AFAIR `akka.http.routing.file-io-dispatcher`)